support python 3.10

## What
support python 3.10

## Why
Currently selene not work with python 3.10 cause of these errors
```bash
(.venv-3.10.0) $ pytest
ImportError while loading conftest '/Users/user/selene/tests/conftest.py'.
tests/conftest.py:7: in <module>
    from selene import Config, Browser
selene/__init__.py:24: in <module>
    from selene.core.entity import (
selene/core/entity.py:1712: in <module>
    from selene.core.conditions import (
selene/core/conditions.py:26: in <module>
    class ElementCondition(Condition[Element]):
../../.pyenv/versions/3.10.0/lib/python3.10/_collections_abc.py:430: in __new__
    raise TypeError(
E   TypeError: Callable must be used as Callable[[arg, ...], result].
(.venv-3.10.0) $
```

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -5,14 +5,17 @@ on: [push, pull_request]
 jobs:
 
   test:
-    name: Tests
     runs-on: ubuntu-20.04
     env:
       DISPLAY: ":99"
-    steps:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [ '3.7', '3.10' ]
 
-      - name: Checkout code
-        uses: actions/checkout@v2
+    steps:
+      - uses: actions/checkout@v2
 
       - uses: technote-space/get-diff-action@v5
         with:
@@ -22,10 +25,10 @@ jobs:
             poetry.lock
             tests.yml
 
-      - name: Set up Python
+      - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
-          python-version: '3.7'
+          python-version: ${{ matrix.python-version }}
         if: env.GIT_DIFF
 
       # can be packaged as Docker-image

